modules/sfp_spur.py

```diff
@@ -19,12 +19,25 @@ class sfp_spur(SpiderFootPlugin):
     """spur.us:Investigate,Passive:Reputation Systems:apikey:Obtain information about any malicious activities involving IP addresses found"""
 
     meta = {
-		'name': "spur.us",
-		'summary': "Obtain information about any malicious activities involving IP addresses found",
-		'flags': [ "apikey" ],
-		'useCases': [ "Investigate", "Passive" ],
-		'categories': [ "Reputation Systems" ]
-	}
+        'name': "spur.us",
+        'summary': "Obtain information about any malicious activities involving IP addresses found",
+        'flags': [ "apikey" ],
+        'useCases': [ "Investigate", "Passive" ],
+        'categories': [ "Reputation Systems" ],
+        'dataSource': {
+            'website': "https://spur.us/",
+            'model': "COMMERCIAL ONLY",
+            'references': [
+                "https://spur.us/api"
+            ],
```

Comment:
Missing instructions?

Reply:
Couldn't go past the paywall, would require your input

Reply:
ok -

1. Pay for a subscription
2. Under the Context API menu on the left, click Tokens
3. Your API key is presented to you

Reply:
Thanks, updated